.github/workflows/tests.yml

```diff
@@ -107,10 +107,12 @@ jobs:
         run: go test -timeout 15m -v ./... -coverprofile=./coverage.txt -covermode=atomic -coverpkg=./pkg...,./cli/...
 
       - name: Upload coverage results to Codecov
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v4
         with:
           fail_ci_if_error: true # if something is wrong on uploading codecov results, then this job will fail
           files: ./coverage.txt
+          slug: nspcc-dev/neo-go
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Could you please add the same token to dbft repo?

Reply:
Like https://github.com/nspcc-dev/dbft/blob/ca4f32856c703ddf46e7d7485f1e6e2c72e6b20d/.github/workflows/go.yml#L49? It was always there even though it's a bit different workflow.

Reply:
Haven't noticed it there.